content_development/MAIN.md

```diff
@@ -51,9 +51,9 @@ Software and technology underpin modern science, which is now almost inevitably
 
 ## What is Open Source Software <a name="What_OSS"></a>
 
-Virtually all modern scientific research workflows rely on a range of software tools, either operating on different datasets, with different parameters, and applied iteratively in various ways (data science) or operating on different inputs and using models and methods to predict some output state (computational science). Open Source Software (OSS) is computer software in which the full source code is available under a specific license that enables other users to access, view, modify, and redistribute that code for any purpose. Typically, as a result of this, it is easier to obtain compared to proprietary software, and re-use for analysis, simulation, and visualisation for research. Because OSS requires such a license, it typically remains free of charge by default.
+Virtually all modern scientific research workflows rely on a range of software tools, either operating on different datasets, with different parameters, and applied iteratively in various ways (data science) or operating on different inputs and using models and methods to predict some output state (computational science). Open Source Software (OSS) is computer software in which the full source code is available under a specific license that enables other users to access, view, modify, and redistribute that code for any purpose. Because OSS requires such a license, it typically remains free of charge by default. Re-using OSS for analysis, simulation and visualisation for research is typically easier and more flexible compared to proprietary software.
```

Comment:
What do you want to say with this sentence?

> Because OSS requires such a license, it typically remains free of charge by default.

Is the difference between open source and free software important?

Reply:
Aha. Yes, let me fix this - great spot.